src/helpers/cauldron/position/getUserPositions.ts

```diff
@@ -0,0 +1,124 @@
+import { Contract, utils, BigNumber, providers } from "ethers";
```

Comment:
In theory, this can all be replaced by the MarketLens, right? Should be inside the positionInfo call.